### ISSUE TITLE
STAS-F15: Insufficient Colour Contrast (Advisory)

### ISSUE THREAD

**Issue by josielewis-warren** [FutureNorthants/northants-website#11]
**Frontend or Backend**: Frontend / Backend / Both

**Description**
The combination of text and background colour should be set to create an easy to read website. Using colours that are similar for the background and foreground can cause blocks of text to become difficult to read. Alternative stylesheets can be used to change the appearance of the page and provide an alternative with a stronger contrast.
The minimum colour contrast ratio between the foreground and background should be at least 4.5:1. For large text, the ratio can be lowered to 3:1.
Text is considered large if it is
- greater than 18 point (approx. 24px) or
- greater than 14 point (approx. 18.6px) if bolded (font-weight:700 or more).
Throughout the site, there are combinations of colours that fall below the minimum contrast levels that make the text difficult to read.

**How to reproduce**
The grey on white on the text that indicate the size of the pdf files on the ‘fees-and-charges’ page, and several other pages throughout the site, fails colour contrast requirements with a ratio of 3.9 and a text size of 14.4px.

**Additional context**
“There are a number of instances where the text is made harder to read because of the colour combination used. Changing either the foreground colour or background colour to make the combination stronger would help low vision and colour blind people read the content.”
https://www.northnorthants.gov.uk/finance/fees-and-charges
https://www.northnorthants.gov.uk/councillors-and-democracy/current-consultations
https://www.northnorthants.gov.uk/benefits/apply-council-tax-support
https://www.northnorthants.gov.uk/your-council/comments-compliments-and-complaints
